package/MDAnalysis/core/universe.py

```diff
@@ -1498,8 +1498,11 @@ def Merge(*args):
                    residue_segindex=segidx)
 
     # Create and populate a universe
-    coords = np.vstack([a.positions for a in args])
-    u = Universe(top, coords[None, :, :],
+    try:
+        coords = np.vstack([a.positions for a in args])
+        u = Universe(top, coords[None, :, :],
                  format=MDAnalysis.coordinates.memory.MemoryReader)
-
+    except:
+        u = Universe(top, format=MDAnalysis.coordinates.memory.MemoryReader)
```

Comment:
You don't need to specify `format=` here as there are no coordinates

Reply:
I have added tests and changelog entry and removed format=.
Thanks for the help.